backend/impl/src/test/java/com/virtuslab/gitmachete/backend/integration/ParentInferenceIntegrationTestSuite.java

```diff
@@ -65,9 +65,10 @@ public ParentInferenceIntegrationTestSuite(String scriptName, String forBranch,
   @SneakyThrows
   public void parentIsCorrectlyInferred() {
     Set<String> managedBranchNames = gitMacheteRepositorySnapshot.getManagedBranches().map(b -> b.getName()).toSet();
-    Option<ILocalBranchReference> result = gitMacheteRepository.inferParentForLocalBranch(managedBranchNames, forBranch);
-    Assert.assertTrue(result.isDefined());
-    Assert.assertEquals(expectedParent, result.get().getName());
+    @Nullable ILocalBranchReference result = gitMacheteRepository.inferParentForLocalBranch(managedBranchNames, forBranch);
+    //it seems that nullable annotation might not be useful in here, as the value has to always be defined.
```

Comment:
`@Nullable` is **not** needed anyway on local variables (as it's auto-inferred by Checker)